### PR TITLE
Fix url to maki icons

### LINKED ISSUE #5360: Icons not displayed in mapbox-vector-tiles example
The icons are no longer available in https://raw.githubusercontent.com/mapbox/maki/mb-pages/renders/

### PR DESCRIPTION
**Pull request by fredj**
fixes #5360

### CODE REVIEW

File: examples/resources/mapbox-streets-v6-style.js [hunk 1]
```diff
@@ -15,8 +15,7 @@ function createMapboxStreetsV6Style() {
     var icon = iconCache[iconName];
     if (!icon) {
       icon = new ol.style.Style({image: new ol.style.Icon({
-        src: '//raw.githubusercontent.com/mapbox/maki/mb-pages/renders/' +
-            iconName + '-12.png'
+        src: 'https://cdn.rawgit.com/mapbox/maki/master/icons/' + iconName + '-15.svg'
```
Comment (ahocevar) [inline examples/resources/mapbox-streets-v6-style.js]: Will the svg icons work fine in IE? Or do we need to add an `imgSize`?

**Comment by ahocevar** [comment]
Thanks @fredj. I added one question. Looks good, please merge when my question is answered/addressed.

File: examples/resources/mapbox-streets-v6-style.js (hunk 1 above)
Comment (fredj) [inline examples/resources/mapbox-streets-v6-style.js]: you're right, `imgSize`is needed for IE. Thanks for the reminder

**Comment by fredj** [comment]
tested in firefox, chrome, ie 11 and ie edge

**Comment by fredj** [comment]
Thanks for the review

**Pull request merged**